PhysicsTools/PatAlgos/python/slimming/offlineSlimmedPrimaryVertices_cfi.py

```diff
@@ -7,3 +7,7 @@
 # this need new input file with scores made at RECO time (PR #8102, #8101), enable before MINIAOD prod in 74x
 #   score = cms.InputTag("offlinePrimaryVertices"), 
 )
+
+offlineSlimmedPrimaryVertices4D = cms.EDProducer("PATVertexSlimmer",
+    src = cms.InputTag("offlinePrimaryVertices4D"),
```

Comment:
_cfi files should not be used to make multiple instances. Use _cff 
or add this into a separate offlineSlimmedPrimaryVertices4D_cfi.py.

Perhaps more importantly, is it OK in this case to skip rekey of the scores (ignored without the ```score``` argument)?

Reply:
Yes, the re-keying is used for association between the packed candidates and the vertices, and a few other downstream things, which are still using only the 3d vertices for the moment now.